doc/load-balancing.md

```diff
@@ -129,10 +129,9 @@ works:
         by the resolver. It asks the balancer for the server addresses to
         use for the server name originally requested by the client (i.e.,
         the same one originally passed to the name resolver).
-        - Note: The `grpclb` policy currently ignores any non-balancer
-          addresses returned by the resolver. However, in the future, it
-          may be changed to use these addresses as a fallback in case no
-          balancers can be contacted.
+        - Note: In `grpclb` policy, the non-balancer addresses returned by
+          the resolver are used as a fallback in case no balancers can be
+          contacted.
```

Comment:
Please add "when the LB policy is started".

Reply:
Done.